bedrock/mozorg/templates/mozorg/home/home-old.html

```diff
@@ -188,12 +188,7 @@ <h2 class="mzp-c-picto-heading">{{ ftl('home-need-help') }}</h2>
 
     <aside class="mzp-c-newsletter">
       <div class="mzp-c-newsletter-image">
-        {{ resp_img(
-          url='img/home/2018/newsletter-graphic.png',
-          srcset={
-            'img/home/2018/newsletter-graphic-high-res.png': '2x'
-          }
-        ) }}
+        {{ high_res_img('img/home/2018/newsletter-graphic.png', {'alt': ''}) }}
```

Comment:
This is throwing an error on the old home page since `high_res_img()` no longer exists